source/common/http/http1/balsa_parser.cc

```diff
@@ -0,0 +1,204 @@
+#include "source/common/http/http1/balsa_parser.h"
+
+#include <algorithm>
+#include <cctype>
+#include <cstdint>
+
+#include "source/common/common/assert.h"
+
+#include "absl/strings/match.h"
+
+using ::quiche::BalsaFrameEnums;
```

Comment:
I am not able to find such guidance in https://google.github.io/styleguide/cppguide.html#Aliases.  It states that "local convenience aliases are fine [...] in .cc files".  May I leave them where they are, especially now that with Ryan's change there is no need for anonymous namespace otherwise?

Reply:
TIL.  Done.